code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1087,7 +1087,7 @@
 /datum/reagent/toxin/acid/fluacid
 	name = "Fluorosulfuric Acid"
 	description = "Fluorosulfuric acid is an extremely corrosive chemical substance."
-	color = "#5050FF"
+	color = "#00FF32"
```

Comment:
Should be teal grey

Reply:
$\color{#339966}&#9724;$ #339966 - is a green with a slight teal undertone